@here/harp-examples/src/hello2_js-bundle.html

```diff
@@ -0,0 +1,83 @@
+        <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0"
+        />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    </head>
+    <style>
+        body {
+            background: #fff;
+            padding: 0;
+            margin: 0;
+            overflow: hidden;
+        }
+    </style>
+    <body>
+        <!--
+            These paths are to be used by clients, when using map bundle from CDN.
+        -->
+        <!--
+            <script src="https://unpkg.com/three/build/three.min.js"></script>
+            <script src="https://unpkg.com/@here/harp.gl/dist/harp.js"></script>
+        -->
+        <!--
+            These paths are used here to be compatible with harp.gl CI.
+        -->
+        <script src="three.min.js"></script>
+        <script src="harp.js"></script>
+
+        <script>
+            const hereCopyrightInfo = {
+                id: "here.com",
+                year: new Date().getFullYear(),
+                label: "HERE",
+                link: "https://legal.here.com/terms"
+            };
+            const xyzAccessToken = "AYlqpxvwl7C8tSVG22lX2lg";
+
+            const map = new harp.Map(document.body, {
+                themeUrl: "resources/berlin_tilezen_base.json",
+                location: [40.6935, -74.009],
+                zoomLevel: 16.9,
+                cameraYaw: 6.3,
+                cameraPitch: 50,
+                dataSources: [
+                    new harp.OmvDataSource({
+                        baseUrl: "https://xyz.api.here.com/tiles/herebase.02",
+                        apiFormat: harp.APIFormat.XYZOMV,
+                        styleSetName: "tilezen",
+                        maxZoomLevel: 17,
+                        authenticationCode: xyzAccessToken,
+                        copyrightInfo: [hereCopyrightInfo]
+                    })
+                ]
+            });
+            const infoElement = document.createElement("div");
+            infoElement.style.position = "absolute";
+            infoElement.style.right = "0";
+            infoElement.style.top = "0";
+            infoElement.style.backgroundColor = "#f0fef1";
+            infoElement.style.zIndex = "400";
+            infoElement.style.padding = "2px";
+            infoElement.style.fontSize = "0.8em";
+            infoElement.style.fontFamily = "sans-serif";
+            document.body.appendChild(infoElement);
+
+            function humanReadable(n) {
+                return n.toPrecision(5).replace(/(\.)?0+$/, "");
+            }
+            map.addEventListener("camera-updated", () => {
+                const location = map.geoCenter;
+                infoElement.innerHTML = `
+                    Lat: ${humanReadable(location.latitude)}<br>
+                    Long: ${humanReadable(location.longitude)}<br>
+                    Zoom: ${humanReadable(map.mapView.zoomLevel)}
+                `;
+            });
```

Comment:
From the GoogleMaps API entry point, the first hello-world example they showcase is even more simple: 
```js
map = new google.maps.Map(document.getElementById('map'), {
      center: {lat: -34.397, lng: 150.644},
      zoom: 8
});
```

For this we would only need to have internal defaults for the datasources and themes parameters:

### DataSources
I wonder if we even need to define the datasources for this top-level component. Likely the base OMV layer is always needed, and a Features layer could be handled internally if we add some custom objects logic. Then we wouldn't need to expose the datasource model.

### Theme URL
Users for a simple level may not be the ones creating or using tailored themes. We could simplify the theme parameter into simply an optional string being `"day"`, `"reducedDay"` or `"night"` and with an internal default.

Reply:
> Theme URL

As noted, it's first draft.
Theme URL should be default, indeed but we don't have effective means to locate default theme yet.
This can be`https://unpkg.com/@here/harp-map-theme@x.y.z/...` but that's problematic, because
we hardcode dependencies into bundle...
Forces users to use particular CDN.

Another option, is to bundle harp.js with all the styles, and assets and fonts, but then this package becomes huge and we may expected problems - at least until we use `unpkg.com` as CDN.

...
This shows, how concept of 
1) forced theme - Map/MapView doesn't work without theme now
2) forced universal theme on MapView, even if it's only datasource who uses it
3) OmvDataSource which doesn't work at all without theme because it's only theme, but also set of mandatory rules to process data ... and it's not attached to DS, but  to MapView

It's all ... at best suboptimal ..